scripts/loadenv.js

```diff
@@ -19,6 +19,7 @@ const { exec } = require('child_process');
 const mergedEnv = { ...dotenv.parsed, ...process.env };
 const command = process.argv
 	.slice(2)
+	.map((arg) => (arg.includes(' ') && `'${arg}'`) || arg)
```

Comment:
Needed as multi-word arguments with quotes are passed into node as a single array element with a space in between. The existence of a space means the argument needs to be re-quoted.

May have some adverse side-effects based on if the argument actually contains its own single quotes...